scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -137,6 +137,11 @@ negative number indicates that session is infinite (value specified in second) *
     /** List of the scripts paths to execute at scheduler start. Paths are separated by a ';'. */
     SCHEDULER_STARTSCRIPTS_PATHS("pa.scheduler.startscripts.paths", PropertyType.LIST),
 
+    /** Path to boot (early starting) microservices (e.g., configuration and iam micro-services). */
+    SCHEDULER_BOOT_MICROSERVICES_PATH("pa.boot.microservices.path", PropertyType.LIST),
+
```

Comment:
SCHEDULER_BOOT_MICROSERVICES_PATH shoud be of type PropertyType.STRING (only one path)

it would be good to have a property for parameters given to the forked JVM. Here you can set by default -Xmx2048M

You can use for this property a type PropertyType.LIST, in that case, parameters can be passed with a separator defined in the code (for example "," or " ")